scripts/migrate.py

```diff
@@ -283,5 +313,22 @@ def _batch_iter(n, iterable):
         yield batch
 
 
+def collect_es_annotation_ids(es):
+    ids = set()
+    annotations = scan(es)
+    for batch in _batch_iter(BATCH_SIZE, annotations):
+        for ann in batch:
+            ids.add(ann['_id'])
+
+    return ids
+
+
+def collect_pg_annotations():
+   ids = set()
```

Comment:
Seeing as this returns annotations, not IDs, I suspect you might as well call `Session.query(Annotation).all()` above and do away with this function.